### ISSUE TITLE
Quest: [51] Secret of the Circle

### ISSUE THREAD

**Issue by NomadicTurtle** [dalaranwow/dalaran-wow#3051]
Issue:
No part of the quest [51] Secret of the Circle works as intended and therefore cannot be completed.

What should happen: 
After finishing the quest [51] Into the Depths, in The Sunken Temple instance, you get the quest [51] Secret of the Circle. After completing [51] Into the Depths, above you stone snakes light up in an order which you need to memorise. [51] Secret of the Circle requires you to activate these stone snakes found in the Hall of Serpents in the now memorised order. Once this is done, an idol should appear behind a boss that appears in the Pit of Refuse below (near the first altar that you completed  [51] Into the Depths at). After defeating the boss, by clicking on the idol should complete the quest.

What happens:
After finishing the quest [51] Into the Depths, no stone snakes light up. The boss that guards the idol that should only appear once the stone snakes have been activated in the correct order, is spawned at the beginning of the instance. Activating the stone snakes in the right order changes nothing and the idol is never spawned, hence the quest can never be completed.
TL;DR The boss that you are supposed to summon is there from the beginning and the quest cannot be completed.

Boss: Atal'alarion <Guardian of the Idol> (couldn't find an id)

Links:
http://www.blizzardguides.com/sunken_temple_instance.html
https://www.youtube.com/watch?v=6bZ_K95bFVE
http://wowwiki.wikia.com/wiki/Quest:Secret_of_the_Circle

**Comment by Golganeth** [comment]
Reported on #375
